src/playercards/CardsThatRedrawTokens.ttslua

```diff
@@ -0,0 +1,59 @@
+local turnOnHelper
+
+function onSave()
+  return JSON.encode(turnOnHelper)
+end
+
+function onLoad(savedData)
+  self.addContextMenuItem("Enable Helper", makeXMLButton)
+
+  if savedData and savedData ~= "" then
+    turnOnHelper = JSON.decode(savedData)
+    if turnOnHelper == true then
+      makeXMLButton()
+    end
+  end
+end
+
+function makeXMLButton()
+  turnOnHelper = true
+  self.clearContextMenu()
+  self.addContextMenuItem("Clear Helper", deleteButton)
+  local xmlTable = {{
+      tag = "Button",
+      attributes = {
+          height = buttonHeight or 450,
+          width = buttonWidth or 1400,
+          rotation = buttonRotation or "0 0 180",
+          scale = "0.1 0.1 1",
+          position = buttonPosition or "0 -55 -22",
+          padding = "50 50 50 50",
+          font = "font_teutonic-arkham",
+          fontSize = buttonFontSize or 250,
+          onClick = "triggerXMLTokenLabelCreation()",
+          color = buttonColor or "#77674DE6",
+          textColor = "White"
+      },
+      value = buttonValue or "Redraw Token"
+    }}
+  if buttonIcon then
+      xmlTable[1].attributes.iconWidth = "400"
+      xmlTable[1].attributes.iconAlignment = "Right"
+      xmlTable[1].attributes.icon = buttonIcon
+  end
+  self.UI.setXmlTable(xmlTable)
+end
+
+function triggerXMLTokenLabelCreation()
+  -- needs to be its own function in order to pass originParams as a table
+    log(originParams)
+  Global.call("activeRedrawEffect", originParams)
+end
+
+-- Delete button
+function deleteButton()
+  self.clearContextMenu()
+  self.addContextMenuItem("Enable Helper", makeXMLButton)
+  self.UI.setXml("")
```

Comment:
How about showing / hiding the XML instead of recreating it? (e.g. create it once in onload (but hidden) and then only show/hide subsequently)